subhub/api/webhooks/stripe/controller.py

```diff
@@ -41,12 +46,18 @@ def run(self):
             StripeChargeSucceededEvent(self.payload).run()
         elif event_type == "subscription.created":
             StripeSubscriptionCreated(self.payload).run()
+        elif event_type == "invoice.finalized":
+            StripeInvoiceFinalized(self.payload).run()
+        elif event_type == "payment_intent.succeeded":
+            StripePaymentIntentSucceeded(self.payload).run()
+        elif event_type == "invoice.payment_failed":
+            StripeInvoicePaymentFailed(self.payload).run()
         else:
             pass
 
 
 def view() -> tuple:
-    logger.info(f"meta {request}")
+    logger.info(f"meta {request.data}")
```

Comment:
wrong logging paradigm; remember the message is going to be capitalized; any data should be passed as a keyword argument

Reply:
addressed in update